.github/workflows/pants.yaml

```diff
@@ -18,23 +18,24 @@ jobs:
   build:
     name: Perform CI Checks
     needs: org-check
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-20.04
     strategy:
       matrix:
+        # The python version to run Pants itself on.
         python-version: [3.9]
     steps:
     - uses: actions/checkout@v3
-    - uses: pantsbuild/actions/init-pants@v1
+    - uses: pantsbuild/actions/init-pants@v2
       # This action bootstraps pants and manages 2-3 GHA caches.
       # See: github.com/pantsbuild/actions/tree/main/init-pants/
       with:
         pants-python-version: ${{ matrix.python-version }}
         # cache0 makes it easy to bust the cache if needed
         # just increase the integer to start with a fresh cache
         gha-cache-key: cache0-py${{ matrix.python-version }}
-        # The Python backend uses named_caches for Pip/PEX state,
+        # The JVM backend uses named_caches for Coursier state,
```

Comment:
Folks who know the JVM backend, is this correct? Are the contents of the named_caches affected by the JVM lockfile, as they are for Python?

Reply:
That's correct.

The coursier cache _also_ contains the JDK itself, but I don't think that it is worth the complexity of trying to differentiate it.

Reply:
I think that's OK, because that is determined by pants.toml? which is already in the cache key.